### DISCUSSION TITLE
Suggested touch monitors?

### DISCUSSION THREAD

**Discussion by phil-barrett** [Jay-Tech/ioSenderTouch#5] [General]
I've been playing around with building an ioSender based "Masso clone".  Given these inexpensive mini PCs and a touch display, it think it's quite possible.  I want to use Win 10 (but support Win 11, too) and a mini PC.  What monitor are you using?

**Comment by Jay-Tech** [comment]
srry, did not see your post guess I did not have notifications turned on.  But I use a really cheap 15.6 touch screen from amazon i got on prime day for 100 us dolars.
https://www.amazon.com/gp/product/B0C5J932TM/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&th=1 
Have used the app on win 10 and win 11 pro.  The main thing with the monior  is it really needs to be 1920x1080 or 16:9 ratio or something really close to that so  the ui  will  fit the screen with IOSender Touch.  Masso looks to be a 4:3 screen  (just guessing by the way it looks in portait)  but you can make a clone of masso  using 1920x1080 just be alittle different  sizing to get everyting to fit.

**Comment by phil-barrett** [comment]
Thanks.  Did not mean a literal clone, just something equivalent.  Pretty sure a fully tricked out system would be no where close to $1300 USD in cost.

**Reply by Jay-Tech** [reply]
For sure my mini PC, monitor, and the card I use for grblhal (flexihal) under 450 maybe under even 400.
